src/listModel.js

```diff
@@ -0,0 +1,17 @@
+export default [
+  {
+    description: 'take out trash',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'wash dishes',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'walk dog',
+    completed: false,
+    index: 2,
+  },
+];
```

Comment:
As the previous code reviewer said, the task object's values for completion are to be updated upon user actions( that is when the todo object is clicked on as been completed, the complete value should return a boolean true). 

Kindly fix it 👍

Reply:
Are you saying that the javascript file must be edited when there is a task change? Maybe I am wrong, please give a better clarification. Because the new list is saved in the local storage and the boolean true on completed is updated. I am unsure what you mean since the local storage already modifies the new list item's completed to true and that new list is what is loaded when the browser refreshes. Please could I get a clearer explanation as to where the problem in my application lies?

Reply:
I am unsure what you mean since the new array is updated to the local storage in my code as were the requirements. Could I get a better explanation of what you mean? Or are u saying I should modify the actual js file when I update the completed field?